sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.netstandard2.0.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.AI
+{
+    public partial class Client
```

Comment:
This name is too vague. You should set the full client name e.g., `ContentSafetyClient`, in your client tsp.

Reply:
All other type names with the client name - options, tests, etc. - should be similarly named.

Reply:
Oh, seems like the actual generated code may be correct. You need to run `eng/scripts/CodeChecks.ps1 contentsafety` to update all this.

Reply:
@heaths I think this problem is related to the bug I encountered when generating sdk project following this [guidance](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/AzureSDKPackage_Setup.md)
What this step produced all have the name Azure.AI instead of Azure.AI.ContentSafety. I manually fixed most of the names and missed this one.
I'm wondering what's your suggestion here? Continue fixing the name manually on my original project or ask someone's help to fix this issue in PS script?
You can reproduce it by running: 
.\eng\scripts\automation\Invoke-TypeSpecDataPlaneGenerateSDKPackage.ps1  -sdkFolder D:\src\mengaims\azure-sdk-for-net\sdk\contentsafety\Azure.AI.ContentSafety -typespecSpecDirectory specification\cognitiveservices\ContentSafety -commit 462e8d9327ea0dfc77e7df76933b34b3d489953e -repo Azure\azure-rest-api-specs-pr
And I pushed the output sdk project [here](https://github.com/mengaims/azure-sdk-for-net/tree/acs/sdk/contentsafety).

Reply:
Fixed manually.